README.rst

```diff
@@ -219,14 +219,14 @@ file for more info.
     :target: https://pypi.org/project/auth0-python/
     :alt: Latest version released on PyPI
 
-.. |coverage| image:: https://codecov.io/gh/auth0/auth0-python/badge.svg
```

Comment:
Why the order change? Makes it a little easier to see the changes if the badge order is the same.

Reply:
Badge order is the same, just matching up the data store order to match how the badges are being displayed. Can revert if needed.

Reply:
Oh sorry, I mis-read the changes. Thank you!